### ISSUE TITLE
Not all orbs are open-source

### ISSUE THREAD

**Issue by Peter-Darton-i2** [circleci/circleci-docs#7537]
# Locations
[Orbs FAQ -> Report an issue with an orb](https://circleci.com/docs/orbs-faq/#report-an-issue-with-an-orb)

# Details
Document states that "All orbs are open source projects"; this isn't correct.

While this is usually be the case for public orbs (I guess a private company _could_ publish a public orb from a private repository but it's unlikely), this is unlikely to be true for private orbs (those are likely to be closed-source).

FYI this (incorrect) assertion that "all orbs are open source" is further compounded by the fact that, after publishing a private orb (with a non-MIT license) using the orb-toolkit orb, when CircleCI lists the whole orb source in the orb registry it incorrectly prefixes the code with an MIT license claim (which is why I was searching for information on orb licensing, which lead me to this document)

TL;DR: Public orbs might be FOSS (but might not be), but private orbs aren't.

# Suggestion

Instead of saying that (all) orbs are open source, specify that *all circleci-authored orbs (and most other public orbs)* are open source.

**Comment by rosieyohannan** [comment]
Thank you for bringing this to our attention. We're working on correcting this so I will update you when the changes are in progress :-D
